lib/DDG/Spice/Arxiv.pm

```diff
@@ -0,0 +1,21 @@
+package DDG::Spice::Arxiv;
+
+# ABSTRACT: Retrieve articles from arxiv.org by identifier
+
+use DDG::Spice;
+
+spice is_cached => 1;
+spice proxy_cache_valid => "200 7d";
+
+spice wrap_jsonp_callback => 1;
+
+spice to => 'https://duckduckgo.com/x.js?u=http://export.arxiv.org/api/query?id_list=$1';
+
+triggers query_lc => qr/^arxiv[:\s]+?([^\s]+)/;
```

Comment:
Sorry let me clarify:

- @jbarrett is using a regex because the query has the potential to contain no spaces (e.g. "arxiv:1010.2163v1")
- I'm saying because of the way we currently define the "words" in a query, using a word trigger instead of regex should actually work here (this case is pretty rare though, so I just wanted to show you a real example of where this is useful)
- Seeing as a lot of IA's are over-triggering because of the way we currently break up the "words" I still think we should go back to separating on spaces only by default, but we should still provide a way to handle the "words" like we currently do

Does that clarify things?